R/s3_methods.R

```diff
@@ -80,131 +88,236 @@
+#' @param ... Not used.
+#'
 #' @export
-#' @noRd
+#' @rdname S3_as.matrix
 as.matrix.DataFrame = function(x, ...) as.matrix(x$to_data_frame(...))
 
 #' @export
-#' @noRd
+#' @rdname S3_as.matrix
 as.matrix.LazyFrame = function(x, ...) as.matrix(x$collect()$to_data_frame(...))
 
+#' Compute the mean
+#'
+#' @param x A [DataFrame][DataFrame_class], [LazyFrame][LazyFrame_class], or
+#' [Series][Series_class]
+#' @param ... Not used.
+#'
 #' @export
-#' @noRd
+#' @rdname S3_mean
 mean.DataFrame = function(x, ...) x$mean()
 
 #' @export
-#' @noRd
+#' @rdname S3_mean
 mean.LazyFrame = function(x, ...) x$mean()
 
 #' @export
+#' @rdname S3_mean
+mean.Series = function(x, ...) x$mean()
+
+#' Compute the median
+#'
+#' @param x A [DataFrame][DataFrame_class], [LazyFrame][LazyFrame_class], or
+#' [Series][Series_class]
+#' @param ... Not used.
+#'
+#' @export
+#' @rdname S3_median
 #' @importFrom stats median
-#' @noRd
 median.DataFrame = function(x, ...) x$median()
 
 #' @export
 #' @importFrom stats median
-#' @noRd
+#' @rdname S3_median
 median.LazyFrame = function(x, ...) x$median()
 
 #' @export
-#' @noRd
+#' @importFrom stats median
+#' @rdname S3_median
+median.Series = function(x, ...) x$median()
+
+#' Compute the minimum value
+#'
+#' @param x A [DataFrame][DataFrame_class], [LazyFrame][LazyFrame_class], or
+#' [Series][Series_class]
+#' @param ... Not used.
+#'
+#' @export
+#' @rdname S3_min
 min.DataFrame = function(x, ...) x$min()
 
 #' @export
-#' @noRd
+#' @rdname S3_min
 min.LazyFrame = function(x, ...) x$min()
 
 #' @export
-#' @noRd
+#' @rdname S3_min
 min.Series = function(x, ...) x$min()
 
+#' Compute the maximum value
+#'
+#' @param x A [DataFrame][DataFrame_class], [LazyFrame][LazyFrame_class], or
+#' [Series][Series_class]
+#' @param ... Not used.
+#'
 #' @export
-#' @noRd
+#' @rdname S3_max
 max.DataFrame = function(x, ...) x$max()
 
 #' @export
-#' @noRd
+#' @rdname S3_max
 max.LazyFrame = function(x, ...) x$max()
 
 #' @export
-#' @noRd
+#' @rdname S3_max
+max.Series = function(x, ...) x$max()
+
+#' Compute the sum
+#'
+#' @param x A [DataFrame][DataFrame_class], [LazyFrame][LazyFrame_class], or
+#' [Series][Series_class]
+#' @param ... Not used.
+#'
+#' @export
+#' @rdname S3_sum
+sum.DataFrame = function(x, ...) x$sum()
+
+#' @export
+#' @rdname S3_sum
+sum.LazyFrame = function(x, ...) x$sum()
+
+#' @export
+#' @rdname S3_sum
+sum.Series = function(x, ...) x$sum()
+
+#' Convert to a vector
+#'
+#' @param x A Polars Series
+#' @param mode Not used.
+#' @export
+#' @rdname S3_as.vector
 as.vector.Series = function(x, mode) x$to_vector()
 
 
-#' as.character for polars Series
-#' @param x Series
-#' @param ... Additional arguments are ignored.
-#' @param str_length an integer. If specified,
-#' utf8 or categorical type Series will be formatted to a string of this length.
-#' @return character vector
-#' @examples
-#' s = pl$Series(c("foo", "barbaz"))
-#' as.character(s)
-#' as.character(s, str_length = 3)
```

Comment:
@etiennebacher These examples must be removed?

Reply:
it was a mistake, I added them back in https://github.com/pola-rs/r-polars/pull/503